runsc/fsgofer/filter/filter.go

```diff
@@ -23,11 +23,16 @@ import (
 
 // Install installs seccomp filters.
 func Install() error {
-	s := allowedSyscalls
-
 	// Set of additional filters used by -race and -msan. Returns empty
 	// when not enabled.
-	s.Merge(instrumentationFilters())
+	allowedSyscalls.Merge(instrumentationFilters())
+
+	return seccomp.Install(allowedSyscalls)
+}
 
-	return seccomp.Install(s)
+// InstallUDSFilters installs the seccomp filters required to let the gofer connect
```

Comment:
I don't think this function installs any filters anymore.

Reply:
Odd.. looking into it

Reply:
Sorry, if I wasn't clear. I meant the name/comment should be updated to match the new code.

Reply:
Oh! Okay. Got it, updating now.

Reply:
Could you rename the function too?